crates/solvers-dto/src/auction.rs

```diff
@@ -24,7 +24,7 @@ pub struct Auction {
 
 #[serde_as]
 #[derive(Debug, Deserialize)]
-#[serde(rename_all = "camelCase", deny_unknown_fields)]
```

Comment:
I think it even makes sense to remove `deny_unknown_fields` altogether given that any update to these fields became more painful when we migrated the solvers to the gnosis organization.